content/learn/book/ecs/_index.md

```diff
@@ -7,211 +7,76 @@ page_template = "book-section.html"
-    for life in query.iter(){
-        dbg!(life);
-    }
-}
-
-// We can request multiple components in our queries by wrapping them in a (A, B) tuple
-// and mutate the components by requesting &mut A rather than &A
-fn regenerate_life(mut query: Query<(&mut Life, &LifeRegen)>){
-    // We can use "destructuring" to unpack our query items into the corresponding types
-    for (mut life, life_regen) in query.iter_mut(){
-        // .0 means "the first field of a tuple struct"
-        life.current += life_regen.0;
-        
-        // We shouldn't let life regeneration heal our units above full!
-        if life.current > life.max {
-            life.current = life.max;
-        }
-    }
-}
-```
-
-Once systems are added to our app, the **runner** takes this information and automatically runs our systems, typically once during each pass of the **game loop** according to the rules defined in the **schedule**.
-
-Bevy's default execution strategy runs systems in parallel.
-Because the **function signature** of each of our systems fully define the data it can access, we can ensure that only one system can change a piece of data at once (although any number can read from a piece of data at the same time).
-
-Systems within the same **stage** are allowed to run in parallel with each other (as long as their data access does not conflict), and are assigned to a thread to perform work as soon as one is free.
-
-```rust
-fn main(){
-    let app = App::new()
-        .add_plugins(MinimalPlugins)
-        // Startup systems run exactly once, when the schedule is first run
-        .add_startup_system(spawn_damaged_player)
-        // Regular systems are run each time the schedule loops
-        .add_system(regenerate_life)
-        // We can use .before and .after to enforce a consistent ordering of our systems
-        .add_system(print_player_life.after(regenerate_life));
-
-    for i in 1..10 {
-        // This runs our app's schedule a single time
-        // When writing real games, you'll typically want to use App::run to loop the schedule indefinitely,
-        // but this method is very valuable for testing and teaching purposes!
-        app.update();
-    }
-}
-```
+In Bevy, game objects are stored as **entities**, whose data is stored as **components**.
+**Systems** operate on this data, modifying the **world** to carry out the behavior that brings your game to life.
+Together, these these form the basis of Bevy's **ECS**, which unsurprisingly stands for ["Entity-Component-System"](https://en.wikipedia.org/wiki/Entity_component_system).
+Let's go over the most important definitions:
+
+- **Entities:** Game objects (either abstract, like a camera, or tangible, like a player character), whose data is stored as components.
+  - The [`Entity`] type is just a simple identifier (like a URL address, a unique name, or a row number in a database): any combination of components can be added to each entity.
```

Comment:
> any combination of components can be added to each entity.

I feel like this can be saved for the components portion, since by this point, readers presumably don't really understand what components are (other than some form of data).